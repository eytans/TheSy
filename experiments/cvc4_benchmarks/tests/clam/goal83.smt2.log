19:38:50 [WARN] Collecting statistics
19:38:50 [WARN]  ex is: zero
19:38:50 [WARN]  ex is: (succ zero)
19:38:50 [WARN]  ex is: (succ (succ zero))
19:38:50 [WARN]  ex is: nil
19:38:50 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:38:50 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:38:50 [WARN] Running exploration without proof mode, but goals were given
19:38:50 [WARN] Proof mode: true
19:38:50 [WARN]  ex is: zero
19:38:50 [WARN]  ex is: (succ zero)
19:38:50 [WARN]  ex is: (succ (succ zero))
19:38:50 [WARN]  ex is: nil
19:38:50 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:38:50 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:38:50 [WARN] Running exploration without proof mode, but goals were given
19:38:50 [WARN] Proof mode: true
19:38:50 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:38:50 [WARN] proved: (rotate ?ts_ph_Nat_0 nil) => nil
19:38:50 [WARN] proved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:39:11 [WARN] generalized case_split proof as: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) -- (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:39:11 [WARN] proved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:39:11 [WARN] proved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:39:11 [WARN] Found all lemmas
