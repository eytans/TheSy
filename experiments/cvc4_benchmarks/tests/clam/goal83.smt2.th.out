19:38:50 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(append nil ?x) => ?x
(append (cons ?x2 ?y) ?z) => (cons ?x2 (append ?y ?z))
(cons ?x2 (append ?y ?z)) => (append (cons ?x2 ?y) ?z)
(len nil) => zero
zero => (len nil)
(len (cons ?x3 ?y)) => (succ (len ?y))
(rotate zero ?x) => ?x
(rotate (succ ?n) nil) => nil
(rotate (succ ?n) (cons ?y4 ?x)) => (rotate ?n (append ?x (cons ?y4 nil)))
(rotate ?n (append ?x (cons ?y4 nil))) => (rotate (succ ?n) (cons ?y4 ?x))
Searcher: (rotate ?autovar7 nil)
Searcher: (rotate ?autovar7 (cons ?autovar3 ?autovar4))
Searcher: (rotate (succ ?autovar5) ?autovar8)
Searcher: (rotate (succ ?autovar6) ?autovar8)
19:38:50 [0m[33m[WARN] [0m ex is: zero
19:38:50 [0m[33m[WARN] [0m ex is: (succ zero)
19:38:50 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:38:50 [0m[33m[WARN] [0m ex is: nil
19:38:50 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:38:50 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:38:50 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:38:50 [0m[33m[WARN] [0mProof mode: true
19:38:50 [0m[33m[WARN] [0m ex is: zero
19:38:50 [0m[33m[WARN] [0m ex is: (succ zero)
19:38:50 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:38:50 [0m[33m[WARN] [0m ex is: nil
19:38:50 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:38:50 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:38:50 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:38:50 [0m[33m[WARN] [0mProof mode: true
19:38:50 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:38:50 [0m[33m[WARN] [0mproved: (rotate ?ts_ph_Nat_0 nil) => nil
19:38:50 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:39:11 [0m[33m[WARN] [0mgeneralized case_split proof as: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) -- (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:39:11 [0m[33m[WARN] [0mproved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:39:11 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:39:11 [0m[33m[WARN] [0mFound all lemmas
done in 20787
